packages/components/src/local/molecules/sortable-list/index.tsx

```diff
@@ -124,6 +124,11 @@ export const SortableList: React.FC<PropTypes> = ({
           }
         }
       }
+      
+      if (!newValue) {
+        newValue = "New Role" 
```

Comment:
Since we refer to "New Role" from two places, could you declare it as a global constant, please?  Then we won't have a logic break if one string gets changed but not the other.

Reply:
Global constant declared for `New Role`